src/dataflow-types/src/client.rs

```diff
@@ -154,9 +154,10 @@ pub enum StorageCommand<T = mz_repr::Timestamp> {
     /// of sources to import.
     RenderSources(
         Vec<(
-            String,
```

Comment:
We should probably introduce a `struct` for this, but I have a hard time coming up with a name :(